app/src/lib/git/diff.ts

```diff
@@ -454,6 +461,64 @@ async function getImageDiff(
   }
 }
 
+async function getLFSImageDiff(
+  repository: Repository,
+  file: FileChange,
+  hunk: DiffHunk
+): Promise<IImageDiff> {
+  let current: Image | undefined = undefined
+  let previous: Image | undefined = undefined
+
+  let previousStr: string = ""
+  let currentStr: string = ""
+
+  for (let i = 0; i < hunk.lines.length; i++) {
+    const element = hunk.lines[i];
+    if (element.type == DiffLineType.Delete) {
```

Comment:
```suggestion
    if (element.type === DiffLineType.Delete) {
```

Prefer absolute equality.

Reply:
[Fix after code review in](https://github.com/desktop/desktop/pull/17686/commits/170bdc50229313f99bb50d599057a1c4eda6317f) https://github.com/desktop/desktop/pull/17686